examples/3d/irradiance_volumes.rs

```diff
@@ -351,12 +351,7 @@ impl AppStatus {
 
         Text::from_section(
             format!(
-                "{}\n{}\n{}\n{}\n{}",
-                CLICK_TO_MOVE_HELP_TEXT,
-                voxels_help_text,
-                irradiance_volume_help_text,
-                rotation_help_text,
-                switch_mesh_help_text
+                "{CLICK_TO_MOVE_HELP_TEXT}\n{voxels_help_text}\n{irradiance_volume_help_text}\n{rotation_help_text}\n{switch_mesh_help_text}"
```

Comment:
This is quite hard to read. Can we put new line literals in strings in Rust?

Reply:
You're right, that was harder to read.

I fixed merge conflicts and [redid this part](https://github.com/bevyengine/bevy/pull/15033/files#diff-742f4b1cc2d88a5e36b1880053dcaa2076ffe154d9588e3e41e65e1b6cf4a75eR354-R358).